apps/frontend/src/libs/components/avatar/avatar.tsx

```diff
@@ -0,0 +1,28 @@
+import styles from "./styles.module.css";
+
+type Properties = {
+	name?: string;
+};
+
+const FIRST_LETTER_INDEX = 0;
+
+const Avatar = ({ name }: Properties): JSX.Element => {
+	let firstLetter = null;
+
+	if (name && name.length > FIRST_LETTER_INDEX) {
+		firstLetter = name[FIRST_LETTER_INDEX];
+	}
+
+	const hasFirstLetter = Boolean(firstLetter);
+
+	// TODO: render avatar image instead of first letter (in case if image present)
```

Comment:
We can remove this comment, it's not like something isn't working right, we can even not add the profile image feature, and then we will only have this simplified icon.
```suggestion
```